Jumpscale/data/nacl/NACL.py

```diff
@@ -185,7 +173,9 @@ def pubkey(self):
     @property
     def signingkey(self):
         if self._signingkey == "":
-            self._signingkey = nacl.signing.SigningKey(self.privkey.encode())
+            encrypted_key = self.file_read_hex(self.path_privatekey)
```

Comment:
can you please modify `path_privatekey` to be than `path_signingkey`, as it no longer stores a private key